requirements/edx/base.txt

```diff
@@ -104,6 +104,7 @@ django-webpack-loader==0.6.0
 django==1.11.15
 djangorestframework-jwt==1.11.0
 djangorestframework-xml==1.3.0  # via edx-enterprise
+django-xforwardedfor-middleware==2.0 
```

Comment:
this file is generated from base.in, look at the comment at the top of the file.  This file should be updated by running `make upgrade` after updating base.in